temoa/extensions/growth_rates/components/growth_capacity.py

```
@@ -0,0 +1,126 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from pyomo.environ import Constraint, quicksum, value
+
+import temoa.components.geography as geography
```

Ruff (0.15.21)

[warning] Use `from temoa.components import geography` in lieu of alias

Replace with `from temoa.components import geography`

(PLR0402)